gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -112,6 +114,8 @@ def extract_freq_info(
         **{
             f"{prefix}_freq": array_exprs["freq"],
             f"{prefix}_faf": faf["faf"],
+            f"{prefix}_grpmax": ht.grpmax,
+            f"{prefix}_fafmax": ht.gen_anc_faf_max,
```

Comment:
you shouldn't need to select the subset grpmax or faf values here, right? maybe the code should have an additional if here to unnest the release-wide values from the exomes, something like
```
grpmax_expr = ht.grpmax
fafmax_expr = ht.gen_anc_faf_max
if prefix == "exomes":
    grpmax_expr = grpmax_expr.gnomad
    fafmax_expr = fafmax_expr.gnomad
```

though you shouldn't need the extra `if` if the genome freq HT has similar stratifications as the exomes HT (I was going off the schema of the current public v3 release)

Reply:
I don't have them in the joint HT if I don't select them.

Reply:
I thought from the release code, we will just select only the joint_*** & top level data_type from combined_faf_ht.

Reply:
So the `grpmax` and `fafmax` structs for exomes also contain values for the `non_ukb` subset. 
```
    'grpmax': struct {
        gnomad: struct {
            AC: int32,
            AF: float64,
            AN: int32,
            homozygote_count: int64,
            gen_anc: str
        },
        non_ukb: struct {
            AC: int32,
            AF: float64,
            AN: int32,
            homozygote_count: int64,
            gen_anc: str
        }
    }
...
    'fafmax': struct {
        gnomad: struct {
            faf95_max: float64,
            faf95_max_gen_anc: str,
            faf99_max: float64,
            faf99_max_gen_anc: str
        },
        non_ukb: struct {
            faf95_max: float64,
            faf95_max_gen_anc: str,
            faf99_max: float64,
            faf99_max_gen_anc: str
        }
    }
```
I believe KC is suggesting that you only pull `gnomad` in stead of both `gnomad` and `non_ukb` which her code example will do. As written, you will get an retain the `non_ukb` struct which we dont need on the combined FAF reference

Reply:
I changed it.

Reply:
thanks! yes, I was suggesting only pulling the `gnomad` values (values for the entire release) rather than keeping both nested structs within the grpmax and fafmax structs